### DISCUSSION TITLE
There was an unknown error. Make sure you are logged-in to chat.openai.com If issue persists, please report it here.

### DISCUSSION THREAD

**Discussion by Willem7hellies** [GorvGoyl/ChatGPT-Writer#42] [Report Bugs and Issues]
I have read all the other threads about resetting browser settings, reinstalling/updating the browser, and reinstalling the extension, but nothing has worked.

**Comment by MikeSBDJ** [comment]
I had this error too

**Reply by MikeSBDJ** [reply]
Note the work email I am installing this for is different to the email login I use for Chat GPT (which I am logged into)

**Reply by Willem7hellies** [reply]
I had the same thought but logged into the same gmail that my Chat GPT subscription is using. Same problem persists with both my work and personal gmail account

**Comment by abhitriloki** [comment]
same issue persist

**Comment by dmitry-nai** [comment]
Same issue

**Comment by abhitriloki** [comment]
> I have read all the other threads about resetting browser settings, reinstalling/updating the browser, and reinstalling the extension, but nothing has worked.

I also did every troubleshooting but still same issue occurred

**Comment by Manhusayn** [comment]
same issue

**Comment by GorvGoyl** [comment]
folks, I have identified the issue... releasing the next version with the fix now.. thank you for reporting it.

**Reply by abhitriloki** [reply]
Do we get the notification once the update is released?

**Reply by GorvGoyl** [reply]
I just submitted it to the chrome extension store and waiting for them to publish it. In order to update once it becomes live, you need to restart your browser or update it from the extensions page chrome://extensions/  

The currrent version is 2.1.2 and the next version with the fix is 2.1.3
https://chrome.google.com/webstore/detail/chatgpt-writer-write-mail/pdnenlnelpdomajfejgapbdpmjkfpjkp

**Reply by GorvGoyl** [reply]
it's live now.. please update from settings or restart the browser

**Reply by saikrishna15** [reply]
Hi @GorvGoyl, I reloaded from chrome://extensions, do we need to restart the browser?  I see issue still persists even after reloading from chrome://extensions.

**Reply by octopusgt** [reply]
Hi Gourav,
First of all thank you for this wonderful tool. I have version 2.1.7 of the ChatGPT Writer extension and Chrome version 109.0.5414.120 (Official Build) (64-bit) on two almost identical computers (also, same extensions, same versions, same Chrome version, etc.). Oddly, on one computer the extension is working properly, whereas on the other computer it is not working at all. I have the same issue reported by everyone else: I visit chatGPT and come back as instructed by the extension every time on both computers [exact message in red is "Cloudflare security check required. Please visit chat.openai.com once and come back.", but on one, when I come back, even if the alert in red is gone, interestingly the "Generate" button does not work (it spins for a fraction of a second and then it simply does not produce any outcome). It's quite a mistery! I have tried to uninstall and reinstall the extension, log out and log in chatGPT, and it seems that the version of the extension is up to date, but nothing works (please check the gif below). Hope you can help us. Thank you!

![chatGPT_writer_problem](https://user-images.githubusercontent.com/124645634/217137536-1961d551-8819-4ff3-9593-cdaae4641b72.gif)

**Reply by fasu-ba** [reply]
> I just submitted it to the chrome extension store and waiting for them to publish it. In order to update once it becomes live, you need to restart your browser or update it from the extensions page chrome://extensions/
> 
> The currrent version is 2.1.2 and the next version with the fix is 2.1.3 https://chrome.google.com/webstore/detail/chatgpt-writer-write-mail/pdnenlnelpdomajfejgapbdpmjkfpjkp

unfortunately, I have still the same error, and in store there is a version of 2.1.11, not 2.1.3. I can't find a way to update it.

**Reply by MarjanBans** [reply]
> it's live now.. please update from settings or restart the browser

"There was an unknown error. Make sure you are logged-in to [chat.openai.com](https://chat.openai.com/)
If issue persists, please [report it here](https://github.com/GorvGoyl/ChatGPT-Writer/discussions/categories/report-bugs-and-issues)." 
![image](https://user-images.githubusercontent.com/128818662/227538049-9776898f-cc6c-4be9-ac14-07b0a87ea43f.png)

**Comment by Willem7hellies** [comment]
Hi there,

Thanks for the update. I will make sure to update the settings or restart
the browser as advised.

Best regards,

Willem Van der Merwe
c *072 836 4205*

e ***@***.***

"Wisdom can be found in the unlikely of places."
Uncle Henry

**Comment by crglass** [comment]
Same here. "There was an unknown error. Make sure you are logged-in to [chat.openai.com](https://chat.openai.com/)
If issue persists, please [report it here](https://github.com/GorvGoyl/ChatGPT-Writer/discussions/categories/report-bugs-and-issues)."

**Comment by Shentia** [comment]
> I have read all the other threads about resetting browser settings, reinstalling/updating the browser, and reinstalling the extension, but nothing has worked.


Also I have this isssue 
There was an unknown error. Make sure you are logged-in to [chat.openai.com](https://chat.openai.com/)
If issue persists, please [report it here](https://github.com/GorvGoyl/ChatGPT-Writer/discussions/categories/report-bugs-and-issues).

**Comment by seattleswimacademy** [comment]
The issue is still persisting.

**Comment by RenaldoVe** [comment]
The issue is still persisting even with the latest version of the extension.

**Comment by Rocciah** [comment]
Still persisting

**Comment by Mrsathlete22** [comment]
Still persisting. Attached a screen shot.  I am logged in, have the newest version 2.1.3 and refreshed my browser.
![report](https://user-images.githubusercontent.com/124304458/216432720-614a0b68-2a16-40d5-8e5e-0ba07718ff0c.png)
![report](https://user-images.githubusercontent.com/124304458/216432763-64618924-9cd1-4adb-ba23-e151c8d143d1.png)

**Comment by MrNegative370** [comment]
Issue persists, assuming that the system is down/overwhelmed. waint for update like others.

**Comment by mcbroftw** [comment]
Happening here too.

**Comment by Kman0024** [comment]
Still have the same problem

**Comment by abhitriloki** [comment]
it was working yesterday after the update, but again its not working

**Comment by Willem7hellies** [comment]
Same here it stopped working on my side

**Comment by GorvGoyl** [comment]
fixed in version v2.1.4.. please wait for it to go live on chrome extension store https://chrome.google.com/webstore/detail/chatgpt-writer-email-writ/pdnenlnelpdomajfejgapbdpmjkfpjkp/

**Reply by GJabadari** [reply]
It's available now and it works. Great Job

**Reply by octopusgt** [reply]
Didn't work here. No idea why.

**Comment by Encrypted73** [comment]
Yay! Works now thanks! I'll make a review on chrome store mentioning it's fixed. :)

**Comment by octopusgt** [comment]
> I have read all the other threads about resetting browser settings, reinstalling/updating the browser, and reinstalling the extension, but nothing has worked.

Same here, but only on one computer. The other works. Same identical browser and extension versions, all up to date. No idea why.

![chatGPT_writer_problem](https://user-images.githubusercontent.com/124645634/217138129-6fd5026c-0a42-4c57-8363-5e2629f598af.gif)

**Reply by gemini-mouse** [reply]
same problems here, but the other computer is ok.

**Comment by MikeSBDJ** [comment]
Do people note if they are using a personal or Workspace Gmail account?

Michael Nicholls

**Reply by gemini-mouse** [reply]
All personal accounts, the same account exactly

**Comment by GorvGoyl** [comment]
If you are using ChatGPT plus and it's not working for you then I need your help. I don't have access to ChatGPT Plus. Please reach out to me at hey@gourav.io and share login credentials so that I can debug the issue from my end. You can change credentials after that.

**Reply by grrvin** [reply]
Can I just Venmo you a few bucks so you can get plus for a month?

**Reply by octopusgt** [reply]
> If you are using ChatGPT plus and it's not working for you then I need your help. I don't have access to ChatGPT Plus. Please reach out to me at [hey@gourav.io](mailto:hey@gourav.io) and share login credentials so that I can debug the issue from my end. You can change credentials after that.

Hi Gourav, not using ChatGPT plus here. Also, Gmail is a personal account. The extension is working on computer 1 (Windows 10 Pro Ver 22H2; build 19045.2486; Chrome Version 109.0.5414.120, 64-bit; ChatGPT Writer version 2.1.7), but not on computer 2 (Windows 10 Pro Ver 21H2; build 19044.2486; Chrome Version 109.0.5414.120, 64-bit; ChatGPT Writer version 2.1.7). Same gmail, same chatGPT user. Hope this helps. Thank you for looking into this!

**Comment by grrvin** [comment]
Plus member. I have also uninstalled the extension and reinstalled it again. I get the same error.

Computer 1:
Current OS: macOS 13.1 (22C65)
Current version of Chrome: Version 109.0.5414.119 (Official Build) (x86_64)

Computer 2:
Current OS: Windows 10 Pro (19045.2486)
Current version of Chrome: Version 109.0.5414.120 (Official Build) (64-bit)

**Comment by GorvGoyl** [comment]
hey folks, I've added better logging capabilities to v2.1.8 to debug issues. If it's not working then open browser devtools and share error details. something like this:

![screenshot 2023-02-09 at 19 08 00@2x](https://user-images.githubusercontent.com/7106086/217808952-d793dc7e-9ea9-47ba-9ad5-1790770f5eec.png)

**Reply by octopusgt** [reply]
Hi Gourav,
Thank you for assisting us! Here is the screenshot you have r
![Screenshot 2023-02-09 104111](https://user-images.githubusercontent.com/124645634/217862144-d6a51afd-8a92-4a96-81da-8739e509b0a6.jpg)
equested. The extension was already updated to v2.1.8. Same behaviour so far (that is the issue persists). Hope this helps. Thank you!

**Reply by octopusgt** [reply]
I tried on computer 1, where the extension works (computer 2 still has the issue, even if it's virtually identical: same chrome version, same extesions, same accounts, etc.), and I get the same data, but no real errors (in red) like in your example.
Here is the screenshot from the computer n.1. I really do not know why 
 it's working only on one machine.
![Screenshot 2023-02-09 104641](https://user-images.githubusercontent.com/124645634/217864869-63f3701f-95f3-4133-9278-d9aeb68d421d.jpg)

**Reply by GorvGoyl** [reply]
hmm no error is shown. I need to know error from service worker console as well. Here are the steps:
 - go to chrome extensions page, spot chatgpt writer and click on "inspect service worker" to open dev console.

**Reply by octopusgt** [reply]
I followed your instructions (chrome://extensions/ --> inspect service worker, which was inactive and now it's active) on computer 1 (where the issue persists) and it opened a new window with DevTools, but there is no data and no issues in it. Am I missing something? Should I keep this window open and then try to use the extension to populate it with useful data? Thank you.

**Reply by octopusgt** [reply]
Oh yes, if I try to use the extension I get error messages. Let me share them with you.

**Reply by octopusgt** [reply]
Here it is (from inspect serviec worker, after trying to generate an answer):

![Screenshot 2023-02-09 105918](https://user-images.githubusercontent.com/124645634/217866954-3ffff22f-b039-41f6-a15a-fefe9591f077.jpg)

**Reply by octopusgt** [reply]
I did the same on computer 2 (where the extension works) and I do not get any error (no issues) from DevTools.
Hope this helps. Thank you!

**Reply by octopusgt** [reply]
I also got this from the DevTools in Gmail:

![Screenshot 2023-02-09 110529](https://user-images.githubusercontent.com/124645634/217868403-e10a8e9b-2890-40a8-8ea2-349d169d0c6a.jpg)

**Reply by octopusgt** [reply]
I have just updated Chrome to version Version 110.0.5481.78 (Official Build) (64-bit) and the issue persists.
Here is a screenshot from DevTools after updating crhome.
![Screenshot 2023-02-09 110821](https://user-images.githubusercontent.com/124645634/217869607-60448e9a-47dc-4ce1-a9e3-0955b934848f.jpg)

**Reply by octopusgt** [reply]
Last bit of info (hope it's useful).
I also tried to uninstall and reinstall the extension AFTER the last Chrome update and then to reboot Chrome.
The issue persists.
No issues listed in DevTools as earlier unless I activate service worker.
Once I open DevTools from service worker to inspect, I get the following.

Every time I try to click on Generate, nothing happens, but the same error is listed in the DevTools (I tried to press it 10 times with no results). Hope this helps. Thank you!
![Screenshot 2023-02-09 111342](https://user-images.githubusercontent.com/124645634/217871300-1f5033b2-8b9e-4056-ab47-97d5aab2bf37.jpg)

**Reply by grrvin** [reply]
This is what I am seeing on my end... 
<img width="1161" alt="Screenshot 2023-02-09 at 11 20 11 AM" src="https://user-images.githubusercontent.com/6585168/217873202-54b1e78c-724e-4891-87bf-f6bded57d1bc.png">
<img width="1166" alt="Screenshot 2023-02-09 at 11 20 23 AM" src="https://user-images.githubusercontent.com/6585168/217873222-e06fa449-f0e4-4a0e-bac8-b48e932634ef.png">

**Reply by octopusgt** [reply]
No errors listed in DevTools from Gmail on my end. Only "Error: cloudflare_error) "background.5fadff2f.js:5" from service worker every time I press "Generate". And basically the button Generate is not responding, and the extension is not working here. Thank you.

**Reply by GorvGoyl** [reply]
could you click on error to expand it and tell me what it says:

![screenshot 2023-02-10 at 15 58 06@2x](https://user-images.githubusercontent.com/7106086/218048138-ceac9d3f-783a-4a9f-9266-c09255cabb02.png)

**Reply by octopusgt** [reply]
Hi Gourav,
The issue persists even with version 2.1.11.
Hope you can help us. Thank you!

**Comment by Darrellau0121** [comment]
<img width="1757" alt="image" src="https://user-images.githubusercontent.com/124855036/217855222-e5e3c6e5-2b71-45fc-9257-8990c091c1bb.png">
<img width="1722" alt="WeChat0544f53a1976412522ed72c7c2256520" src="https://user-images.githubusercontent.com/124855036/217855548-708cdc98-4cf9-43fc-9037-212323af76b8.png">

**Comment by MikeSBDJ** [comment]
> I have read all the other threads about resetting browser settings, reinstalling/updating the browser, and reinstalling the extension, but nothing has worked.

All working on this end now (after not working for the last few days)

**Comment by vesector** [comment]
Getting the same error, version 2.1.8 and using ChatGPT Plus.

**Comment by wi196324** [comment]
Only ChatGPT Plus users will get an error when using this plugin.

**Comment by GorvGoyl** [comment]
Thanks for reporting. I've fixed the chatgpt plus issue in v2.1.9. It should be live within a day and then you can update the extension or restart the browser.

https://chrome.google.com/webstore/detail/chatgpt-writer-email-writ/pdnenlnelpdomajfejgapbdpmjkfpjkp/

**Reply by grrvin** [reply]
Yep. Working for me now.
